### ISSUE TITLE
pehkui and identity doesn't work together

### ISSUE THREAD

**Issue by dagsfhghgdsgdhf** [Draylar/identity#377]
when you use /scale set pehkui: only the model, movement, jumpforce works mean while the hitbox and eye height don't work
btb didn't test the other ones and if it is different fore you then ok

**Comment by HyperSoop** [comment]
exactly. It seems like Identity applies a forced override for those options
